yarn-project/txe/src/oracle/txe_oracle.ts

```diff
@@ -847,11 +847,13 @@ export class TXE implements TypedOracle {
 
     await this.addUniqueNoteHashesFromPublic(noteHashes);
 
-    this.addPublicLogs(
-      result.avmProvingRequest.inputs.output.accumulatedData.publicLogs.filter(
-        log => !log.contractAddress.equals(AztecAddress.ZERO),
-      ),
-    );
+    // this.addPublicLogs(
+    //   result.avmProvingRequest.inputs.output.accumulatedData.publicLogs.filter(
+    //     log => !log.contractAddress.equals(AztecAddress.ZERO),
+    //   ),
+    // );
+
+    this.addPublicLogs(result.avmProvingRequest.inputs.output.accumulatedData.publicLogs);
```

Comment:
?

Reply:
Ah, a very good point indeed. Lingering debugging attempt from this afternoon, when I merged w/ master. Reverted.